src/main/java/com/github/mc1arke/sonarqube/plugin/ce/pullrequest/AnalysisDetails.java

```diff
@@ -106,13 +107,16 @@ public String getCommitSha() {
     public String createAnalysisSummary(FormatterFactory formatterFactory) {
 
         BigDecimal newCoverage =
-                findQualityGateCondition(CoreMetrics.NEW_COVERAGE_KEY).map(QualityGate.Condition::getValue)
-                        .map(BigDecimal::new).orElse(null);
+                findQualityGateCondition(CoreMetrics.NEW_COVERAGE_KEY)
+                    .filter(condition -> condition.getStatus() != EvaluationStatus.NO_VALUE)
+                    .map(QualityGate.Condition::getValue)
+                    .map(BigDecimal::new).orElse(null);
 
         double coverage = findMeasure(CoreMetrics.COVERAGE_KEY).map(MeasureWrapper::getDoubleValue).orElse(0D);
 
         BigDecimal newDuplications = findQualityGateCondition(CoreMetrics.NEW_DUPLICATED_LINES_DENSITY_KEY)
-                .map(QualityGate.Condition::getValue).map(BigDecimal::new).orElse(null);
+            .filter(condition -> condition.getStatus() != EvaluationStatus.NO_VALUE)
```

Comment:
Since this is basically the same code as above, I think it would be nice if it would be in the same code format

Reply:
sure, I'd reformat these fragments.